draft-ietf-core-pubsub.md

```diff
@@ -368,8 +368,9 @@ Client                                          Broker
 
 ## CREATE
 
-A CoAP pubsub broker MAY allow Clients to create new topics on the 
-broker using CREATE. A client wishing
+A CoAP pubsub broker SHOULD allow Clients to create new topics on the 
+broker using CREATE. Some exceptions are for fixed brokerless devices
```

Comment:
How about: "Fixed brokerless devices and pre-configured brokers in dedicated installations may disallow using CREATE".